client/jetpack-cloud/sections/partner-portal/types.ts

```diff
@@ -0,0 +1,6 @@
+export enum LicenseStates {
+	All = 'all',
+	Detached = 'detached',
+	Attached = 'attached',
+	Revoked = 'revoked',
+}
```

Comment:
I suggest we keep license filters and license states as separate enums as they serve a different purpose (even if they are almost identical, actually exactly because they are almost identical :thonking:). For example, with this change LicenseStates.All is now technically a possible state for a license to be in but it's not a valid state.
I'd also suggest we keep the naming in singular form as [that seems to be the convention](https://www.typescriptlang.org/docs/handbook/enums.html).
```suggestion
export enum LicenseState {
	Detached = 'detached',
	Attached = 'attached',
	Revoked = 'revoked',
}

export enum LicenseFilter {
	All = 'all',
	Detached = 'detached',
	Attached = 'attached',
	Revoked = 'revoked',
}
```